crates/runtime/src/lib.rs

```diff
@@ -546,10 +550,20 @@ impl Runtime {
 
         // Shutdown signal
         let shutdown_signal_future = async {
-            shutdown_signal().await;
-            tracing::debug!("Shutdown signal received.");
-            self.shutdown().await;
-            Ok(())
+            let graceful_shutdown = async {
+                shutdown_signal().await;
+                tracing::debug!("Shutdown signal received.");
```

Comment:
Maybe inform about force shutdown option?

```suggestion
                tracing::debug!("Shutdown signal received. Press Ctrl-C again to force exit.");
```